### ISSUE TITLE
Have you tried using a bayesian classifier?

### ISSUE THREAD

**Issue by c4milo** [ryanuber/go-license#3]
This package is great, I have a similar need and I was wondering if you tried  using a Bayesian classifier for this.

**Comment by client9** [comment]
Hi @c4milo -- where you encounting problems with the existing code?  Perhaps you could open a pull request of license files that *weren't detected* in perhaps `fixtures/variants`   thanks all!

**Comment by ryanuber** [comment]
@client9 I don't think this is really an issue, but a thought ticket on perhaps a better way to do license scanning rather than the naive full-text scan we do currently. I explored this a bit, but got into the weeds when trying to distinguish similar licenses, like the *GPL's or *BSD's. I might revisit this at some point, and I think there are probably some easy performance wins we can get even with the current code.
